src/Utilities.h

```diff
@@ -60,6 +60,35 @@ inline int u32toa(uint32_t value, char *dst) {
     return ret;
 }
 
+inline bool asciiEqualsCaseI(const char * str1, const char * str2, size_t num)
```

Comment:
This could probably be more concise e.g:

```cpp
bool iequals(const string& a, const string& b)
{
    return std::equal(a.begin(), a.end(),
                      b.begin(), b.end(),
                      [](char a, char b) {
                          return tolower(a) == tolower(b);
                      });
}
```
(unashamedly taken from https://stackoverflow.com/a/4119881)